admin/server/reports.go

```diff
@@ -488,6 +503,23 @@ func (s *Server) yamlForManagedReport(opts *adminv1.ReportOptions, ownerUserID s
 	res.Annotations.AdminNonce = time.Now().Format(time.RFC3339Nano)
 	res.Annotations.WebOpenPath = opts.WebOpenPath
 	res.Annotations.WebOpenState = opts.WebOpenState
+	switch opts.WebOpenMode {
+	case adminv1.ReportOptions_OPEN_MODE_RECIPIENT, adminv1.ReportOptions_OPEN_MODE_UNSPECIFIED: // backwards compatibility
+		res.Annotations.WebOpenMode = WebOpenModeRecipient
```

Comment:
1. Is `web_open_mode` maybe becoming a bit of a misnomer? It doesn't sound like it dictates the access tokens being generated. Can you think of a clearer name? Maybe something like `security_mode`, `token_mode`, or just `open_mode`?
2. Should these modes perhaps apply both to the download and the open link? E.g. if the mode is "recipient", I wonder if both the download and open links should use recipient attributes for consistency?

Reply:
1. `web_open_mode` is just dictating how and if open link is generated [here](https://github.com/rilldata/rill/pull/6216/files#diff-c2bcafed81fcbf8eb2837df22cb63d055009c3c64d5e9d354fbc41659e7b36aeR103-R106) so seems like appropriate name?
2. In the last change we made sure that all recipients use magic token to download link (using creators' attribute if there is an owner) then if we change open mode behaviour then that will change this behaviour for `recipient` mode and thus they won't be able to download if they don't have access.

Reply:
we decided to go with current approach of using `recipient` mode for now as default and once `filtered` mode is implemented we will just use that.